genai-perf/genai_perf/wrapper.py

```diff
@@ -152,7 +152,12 @@ def run(
     ) -> None:
         try:
             if telemetry_data_collector is not None:
-                telemetry_data_collector.start()
+                if telemetry_data_collector.check_url_reachability():
+                    telemetry_data_collector.start()
+                else:
+                    logger.warning(
+                        f"The metrics url ({telemetry_data_collector.metrics_url}) is unreachable, cannot collect telemetry data"
```

Comment:
A little too long. Also a run-on sentence. We might want to make it shorter.
Something like:
```
The metrics url ({telemetry_data_collector.metrics_url}) is unreachable.
GenAI-Perf cannot collect telemetry data.
```